playbooks/group_vars/xnat.yml

```diff
@@ -80,19 +80,18 @@ firewalld_work_zone_open_services:
   - https
 
 firewalld_internal_zone_ports:
-  - "{{ nginx_upstream_listen_port }}"
+  - "{{ default_dicom_listen_port }}"
 
 firewalld_work_zone_ports:
-  - "{{ nginx_upstream_listen_port }}"
+  - "{{ default_dicom_listen_port }}"
 
 firewalld_public_zone_ports:
-  - "{{ nginx_upstream_listen_port }}"
+  - "{{ default_dicom_listen_port }}"
 
 # mirsg.infrastructure.nginx
 nginx_use_ssl: "{{ ssl.use_ssl }}"
 nginx_server_name: "{{ web_server.host }}"
-nginx_upstream_port: 8104
-nginx_upstream_listen_port: 8104
+default_dicom_listen_port: 8104
```

Comment:
perhaps rename to `nginx_default_dicom_listen_port` (as ansible-lint likes it when variables being with the role name)

Reply:
Ah, yeah - good point, thanks!

Reply:
FYI I've made the change simpler and more generic by retaining the old `nginx_upstream_listen_port` name

Reply:
good idea! Will make things easier